targets/javascript/make.js

```diff
@@ -97,12 +106,21 @@ function hasResultActions(apiCall) {
 function getResultActions(tabbing, apiCall) {
     if (apiCall.result === "LoginResult")
         return tabbing + "if (result != null) {\n"
-            + tabbing + "       if(result.data.SessionTicket != null) {\n"
-            + tabbing + "           PlayFab._internalSettings.sessionTicket = result.data.SessionTicket;\n"
-            + tabbing + "       }\n"
-            + tabbing + "       if (result.data.EntityToken != null) {\n"
-            + tabbing + "           PlayFab._internalSettings.entityToken = result.data.EntityToken.EntityToken;\n"
-            + tabbing + "       }\n"
+            + tabbing + "   // Capture the playFabId, and make sure the authenticationContext exists with the ID in question\n"
+            + tabbing + "   var playFabId = result.data.PlayFabId;\n"
+            + tabbing + "\n"
+            + tabbing + "   if(PlayFab._internalSettings.authenticationContext.hasOwnProperty(playFabId) == false) {\n"
+            + tabbing + "       PlayFab._internalSettings.authenticationContext[playFabId] = {};"
+            + tabbing + "   }\n"
+            + tabbing + "\n"
+            + tabbing + "   if(result.data.SessionTicket != null) {\n"
+            + tabbing + "       PlayFab._internalSettings.sessionTicket = result.data.SessionTicket;\n"
+            + tabbing + "       PlayFab._internalSettings.authenticationContext[playFabId]['sessionTicket'] = result.data.SessionTicket;\n"
```

Comment:
Use .variableName when appropriate.
For example, when accessing variables on an authenticationContext, the syntax should be written as:
authenticationContext.sessionTicket
not
authenticationContext['sessionTicket']